src/main/external-resources/renderers/Panasonic-VieraE6.conf

```diff
@@ -52,12 +50,12 @@ Supported = f:mpegts        v:mp4                      a:aac-lc|he-aac|ac3|eac3|
 Supported = f:wmv|asf       v:vc1                      a:aac-lc|he-aac|ac3|eac3|lpcm|mp3|wma                m:video/x-ms-wmv
 
 # Supported audio formats:
-Supported = f:3gp|3g2|mp4   m:audio/mp4                a:aac-lc|he-aac
-Supported = f:m4a           m:audio/x-m4a              a:(?!alac).+
+Supported = f:3ga|3g2a|m4a  m:audio/mp4                a:aac-lc|aac-ltp|he-aac|he-aacv2
+Supported = f:m4a           m:audio/x-m4a              a:(?!aac-|he-aac)
 Supported = f:adts          m:audio/vnd.dlna.adts      a:aac-lc|he-aac
 Supported = f:flac          m:audio/flac
 Supported = f:mp3           m:audio/mpeg
-Supported = f:wav           m:audio/x-wav              a:dts|lpcm                s:48000
+Supported = f:wav           m:audio/wav                a:dts|lpcm                s:48000
```

Comment:
Yes, it's mine ;)
I can also tell you that the users are not able to get WAV since some tile already.
That's said it was working at first.